test/functional/apps/visualize/_tile_map.js

```diff
@@ -75,10 +74,6 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.visualize.clickBucket('Geo Coordinates');
         log.debug('Click aggregation Geohash');
         await PageObjects.visualize.selectAggregation('Geohash');
-        log.debug('Click field geo.coordinates');
-        await retry.try(async function tryingForTime() {
-          await PageObjects.visualize.selectField('geo.coordinates');
```

Comment:
If more fields are added to configuration in the future, `geo.coordinates` will not be selected automatically, so we cannot rely on auto selection here. We need to update `selectField` function to work with `auto_select_if_only_one` logic

Reply:
done
@dmlemeshko please, take a look